### ISSUE TITLE
Fails to run on Solus, Unknown package manager, GLIBC_2.33 not found 

### ISSUE THREAD

**Issue by equareo** [ubports/ubports-pdk#23]
**git clone https://github.com/ubports/ubports-pdk**

**./ubuntu-touch-pdk setup**

WARNING: You haven't set up your environment yet. Continuing with setup...
Installing prerequisites
snap "qemu-ut-pdk" is already installed, see 'snap help refresh'
KVM group all set up, good.
Please create and enter the directory path you want to set up.
This directory will contain your VM images and source code.
Path: ubports
WARNING: The OpenSSH server seems to be missing or not activated, please install it using your package manager.
Would you like to do that automatically now [y/n]? 
Unknown package manager used, please add support for it in UBports PDK.  **<- Solus uses eopkg**
Inserting ssh key into authorized keys list
Generating image with settings
100+0 records in
100+0 records out
104857600 bytes (105 MB, 100 MiB) copied, 0.838064 s, 125 MB/s
mkfs.fat 4.1 (2017-01-24)
Copying settings to image
Using mtools to create a settings image

**./ubuntu-touch-pdk pull**

Pull successful!

**./ubuntu-touch-pdk run**

Name of the environment: default

WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
qemu-system-x86_64: warning: dbind: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-Z0DWnohoda: No such file or directory
Couldn't open libEGL.so.1: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /var/lib/snapd/lib/gl/libEGL.so.1)

_**OpenSSH server and Glibc_2.33 is already installed**_

**Comment by thebeardedpapi** [comment]
Did you ever find a solution?

I get the same thing on fedora

```
❯ ./ubuntu-touch-pdk run
#####################################
Ubuntu Touch Platform Development Kit
#####################################

Executing tasks:
	Setup? no
	Clear cache? no
	List? no
	Pull image? no
	Run an image? yes


Name of the environment: default

WARNING: cgroup v2 is not fully supported yet, proceeding with partial confinement
Gtk-Message: 21:54:25.277: Failed to load module "pk-gtk-module"
Gtk-Message: 21:54:25.278: Failed to load module "pk-gtk-module"
Couldn't open libEGL.so.1: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /var/lib/snapd/lib/gl/libEGL.so.1)
```

however i am on `glibc-2.34-7` - I'm going to try downgrading to 2.33, maybe i'll have better luck.

**Comment by fredldotme** [comment]
I can only assume here: Are you using NVIDIA graphics cards with proprietary drivers?
If yes then that might be a problem.

**Comment by equareo** [comment]
yes i have the nvidia proprietary driver. I didn't find a solution.

**Comment by thebeardedpapi** [comment]
I am @fredldotme . NVIDIA card has always been a pain in the ass on linux.
